R/caching.R

```diff
@@ -152,10 +153,13 @@ get_hashed_repo_to_dir_mapping <- function(local_repos) {
 #'   (0: None, 1: packages that get installed + high-level git operations,
 #'   2: includes git checkout infos)
 #'
-#' @return the upstream and/or downstream dependency graphs
-rec_checkout_repos <- function(repos_to_process, feature, direction = c("upstream"),
-                               local_repos = get_local_pkgs_from_config(),
-                               verbose = 0) {
+#' @return A list, on entry per checkout out repository whose name is the name of the
+#'   repo in the form x @@ host and whose value is the directory to
+#'   which it has been checked out into
+get_internal_dependencies <- function(repos_to_process, feature,
```

Comment:
I would rename this back to `rec_checkout_repos` because it has a significant side effect.

Reply:
renamed to `rec_checkout_internal_deps`